locales/sv_SE/pcsx2_Main.po

```diff
@@ -1208,14 +1206,12 @@ msgid "Welcome to PCSX2!"
 msgstr "Välkommen till PCSX2!"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:106
-#, fuzzy
 msgid "Configuration Guide"
-msgstr "KonfigurationsVägledningen (uppkopplat)"
+msgstr "KonfigurationsVägledningen"
```

Comment:
'Guide' is a perfectly normal Swedish word and there's no reason not to use it here.

Reply:
Yes "Guide" is officially in (modern) swedish, but not true swedish as "Vägledning" is - don't you agree? So why not use "Vägledning" when it fits in?